src/stlink-lib/sg.h

```diff
@@ -9,9 +9,6 @@
 #include <stlink.h>
 #include <libusb_settings.h>
 
-#ifdef __cplusplus
-extern "C" {
-#endif
 
```

Comment:
```suggestion
```

Reply:
Addressed in commit da3d9e3f1e1a6d116df5830cf506f30ff56196b3.